IPython/core/display.py

```diff
@@ -182,14 +209,68 @@ def display(*objs, **kwargs):
             if metadata:
                 # kwarg-specified metadata gets precedence
                 _merge(md_dict, metadata)
-            publish_display_data(data=format_dict, metadata=md_dict,
-            **kwargs)
+            publish_display_data(data=format_dict, metadata=md_dict, **kwargs)
+    if display_id:
+        return DisplayHandle(display_id)
+
 
+def update_display(obj, *, display_id, **kwargs):
```

Comment:
It is in the docstring, in that `*` is Python's syntax to denote the end of positional args and `display_id` is labeled as keyword-only in the docstring. What would you recommend for comments regarding adopting new Python syntax?